prompt2model/utils/logging_utils.py

```diff
@@ -14,10 +14,14 @@ def get_formatted_logger(logger_name: str):
         A logger object.
     """
     logger = logging.getLogger(logger_name)
-    ch = logging.StreamHandler()
-    formatter = logging.Formatter(
-        "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
-    )
-    ch.setFormatter(formatter)
-    logger.addHandler(ch)
+    # Check if the logger already has a StreamHandler to prevent adding another
```

Comment:
Make this a complete sentence:
```suggestion
    # Check if the logger already has a StreamHandler to prevent adding another.
```